beets/autotag/match.py

```diff
@@ -233,6 +233,11 @@ def distance(items, album_info, mapping):
         dist.add_string('catalognum', likelies['catalognum'],
                         album_info.catalognum)
 
+    # Discogs Catalog number.
+    if likelies['discogs_catalog'] and album_info.catalognum:
+        dist.add_string('catalognum', likelies['discogs_catalog'],
+                        album_info.catalognum)
+
```

Comment:
I am not sure about this and would need advice from @wisp3rwind or @sampsyo. Do I overwrite something here? We use album_info.catalognum in the block before when handling the regular catalognum field.

I am also not yet sure what should be taken into account when both discogs_catalog and catalognum tags exist on a file to be imported.